lib/provisioning.js

```diff
@@ -81,52 +83,54 @@ function detect(options, callback) {
 				distribution: 0
 			},
 
-			ppRegExp = /.*\.(mobileprovision|provisionprofile)$/;
-			
+			ppRegExp = /.*\.(mobileprovision|provisionprofile)$/;			
```

Comment:
Weird trailing whitespace...

```suggestion
			ppRegExp = /.*\.(mobileprovision|provisionprofile)$/;
```

Reply:
Interesting, that come from the indentation of the new loop that surrounds it. I assume we don't have a proper prettier setup for this project, so it wasn't auto-fixed.

Reply:
I think `master` has something, but `1_7_X` branch is too old.